placementApp/views.py

```diff
@@ -88,22 +96,21 @@ def perform_create(self, serializer):
 
 
 class PositionViewSet(viewsets.ModelViewSet):
-    permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
+    permission_classes = (IsStaffOrReadOnly,)
     queryset = Position.objects.all()
+
     def get_serializer_class(self):
-        if self.action in ['list', 'retrieve']:
+        if self.action in ["list", "retrieve"]:
             return PositionReadSerializer
-        print("here")
         return PositionWriteSerializer
 
+
 class CompanyViewSet(viewsets.ModelViewSet):
     permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
     queryset = Company.objects.all()
     serializer_class = CompanySerializer
 
 
-    
-
 def get_xls(request, company_id):
```

Comment:
We need custom permissions for generating excel sheets also

Reply:
Only tpo can generate the excel sheet?